Assets/Scripts/Utils/NameGenerator.cs

```diff
@@ -0,0 +1,37 @@
+using System.Text;
+using UnityEngine;
+
+
+public class NameGenerator : MonoBehaviour
+{
+    [Header("NOTE: This script can be added for each type of thing that needs to be named!")]
+    [Header("Word Lists")]
+    [SerializeField]
+    [Tooltip("The comma-separated list of descriptors to generate names from.")]
+    private TextAsset listOfAdjectives;
+    [SerializeField]
+    [Tooltip("The comma-separated list of nouns to generate names from.")]
+    private TextAsset listOfNouns;
+
+    public string GenerateName()
+    {
+        // Get random name
+        StringBuilder generatedName = new();
+        generatedName.Append(GetRandomAdjectiveFromFile());
```

Comment:
Can you please make this adjustable so we can have a space optionally between the words (for example in the soul names)